### PR TITLE
Support AlmaLinux in Vulnerability Detector

### PR DESCRIPTION
**Pull request by mateocervilla**
|Related issue|
|---|
|#5901|

## Description

This PR adds the new support implemented for AlmaLinux in Vulnerability Detector. 
This change must be applied with the changes performed in wazuh/wazuh#10774.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

**Comment by lexzz** [comment]
@MarcelKemp When will this feature be available in Wazuh? Thank you in advance!

**Comment by MarcelKemp** [comment]
Hi @lexzz, Vulnerability Detector support for AlmaLinux is planned to be available for Wazuh **`v4.5`**.

File: source/user-manual/capabilities/vulnerability-detection/offline-update.rst [hunk 1]
```diff
@@ -362,6 +362,41 @@ Alternatively, they also can be loaded from a local path as follows:
         <update_interval>1h</update_interval>
     </provider>
 
+AlmaLinux
+---------
+
+Currently, the AlmaLinux vulnerabilities are fetched from one OVAL file for each supported AlmaLinux version.
```
Comment (javimed) [inline source/user-manual/capabilities/vulnerability-detection/offline-update.rst:368]:
```suggestion
Currently, the module fetches AlmaLinux vulnerabilities from one OVAL file for each supported AlmaLinux version.
```

File: source/user-manual/capabilities/vulnerability-detection/offline-update.rst [hunk 2]
```diff
@@ -362,6 +362,41 @@ Alternatively, they also can be loaded from a local path as follows:
...
+
++-------------------+------------------------------------------------------------------------------------------------+
+| OS                | Files                                                                                          |
++===================+================================================================================================+
+| AlmaLinux 8       | `org.almalinux.alsa-8.xml <https://security.almalinux.org/oval/org.almalinux.alsa-8.xml>`_     |
++-------------------+------------------------------------------------------------------------------------------------+
+| AlmaLinux 9       | `org.almalinux.alsa-9.xml <https://security.almalinux.org/oval/org.almalinux.alsa-9.xml>`_     |
++-------------------+------------------------------------------------------------------------------------------------+
+
+Feeds can be fetched from a custom repository using the `url` attribute for each supported OS, as shown in this example:
```
Comment (javimed) [inline source/user-manual/capabilities/vulnerability-detection/offline-update.rst:378]:
```suggestion
To fetch feeds from a custom repository you can use the ``url`` attribute for each supported OS, as shown in this example:
```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by javimed — APPROVED** [review]

**Pull request merged**